praw/models/reddit/subreddit.py

```diff
@@ -306,6 +307,22 @@ def wiki(self):
             self._wiki = SubredditWiki(self)
         return self._wiki
 
+    @property
+    def emoji(self):
+        """Provide an instance of :class:`.SubredditEmoji`.
+
+        This attribute can be used to discover all emoji for a subreddit:
+
+        .. code:: python
+
+           for emoji in reddit.subreddit('iama').emoji:
+               print(emoji)
+
+        """
+        if self._emoji is None:
+            self._emoji = SubredditEmoji(self)
+        return self._emoji
+
     def __init__(self, reddit, display_name=None, _data=None):
         """Initialize a Subreddit instance.
```

Comment:
Can you put `self._emoji = None` somewhere in `__init__`? I've cloned your code, and I get `AttributeError: 'Subreddit' object has no attribute 'emoji'` when I try to access `reddit.subreddit('mysub').emoji`.